lambdas/tests/unit/services/test_dynamo_query_service.py

```diff
@@ -91,19 +93,29 @@ def test_lambda_handler_returns_items_from_dynamo(mock_dynamo_table, mock_boto3_
 def test_error_raised_when_no_fields_requested(mock_dynamo_table, mock_boto3_dynamo):
     with patch.object(boto3, "resource", return_value=mock_boto3_dynamo):
         mock_boto3_dynamo.Table.return_value = mock_dynamo_table
-        query_service = DynamoQueryService("test_table", "test_index")
+        query_service = DynamoDBService(
+            "test_table",
+        )
         with pytest.raises(InvalidResourceIdException):
-            query_service("NhsNumber", "0123456789", [])
+            query_service.query_service("test_index", "NhsNumber", "0123456789", [])
```

Comment:
Should this be called DynamoDbServiceMock to avoid calling query_service.query_service?

Reply:
Thanks. We renamed the mock dynamodb service to `db_service`, as Noga did in some other tests